CS342MemberViews/Sources/DannyPark/MapView.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  MapView.swift
+//  Landmarks
+//
+//  Created by Danny Park on 1/22/23.
+//
+
+import SwiftUI
+import MapKit
```

Comment:
SwiftLint wants imports to be sorted in alphabetical order, like this:

```suggestion
import MapKit
import SwiftUI
```